invenio_app_ils/ui/src/invenio_app_ils/pages/backoffice/ItemsSearch/ItemsSearch.js

```diff
@@ -108,6 +58,7 @@ export class ItemsSearch extends Component {
             text={'New item'}
             clickHandler={() => {
               // TODO: EDITOR, implement create form
+              console.log('Implement me');
```

Comment:
I think you can remove the console.log, we marked with TODO all these ones @zzacharo is normally aware.